bread/layout/components/tooltip.py

```diff
@@ -0,0 +1,268 @@
+from typing import Optional, Union
+
+import htmlgenerator as hg
+
+from bread.layout.components.button import Button
+from bread.layout.components.icon import Icon
+from bread.utils import Link
+
+
+def _get_icon(icon):
+    """Return a proper Icon instance"""
+    if isinstance(icon, str):
+        icon = Icon(icon, size="16")
+    else:
+        icon.attributes["width"] = "16"
+        icon.attributes["height"] = "16"
+
+    return icon
+
+
+class DefinitionTooltip(hg.BaseElement):
+    """Definition tooltip is for regular use case of tooltip, e.g. giving the user more
+    text information about something, like defining a word.
+
+    This works better than the interactive tooltip in regular use cases because
+    the info icon used in interactive tooltip can be repetitive when it’s shown several times
+    on a page. Definition tooltip does not use any JavaScript.
+
+    Reference: https://the-carbon-components.netlify.app/?nav=tooltip
+    """
+
+    def __init__(
+        self,
+        label: str,
+        description: str,
+        align: str = "center",
+        position: str = "bottom",
+        **attributes,
+    ):
+        """
+        Parameters
+        ----------
+        label : str
+            Text to be displayed
+        description : str
+            One line of string defining the label
+        align : str, optional
+            Specify where the arrow pointing the icon should align to the text.
+            It can be either start, center, or end.
+            The default value is 'center'.
+        position : str
+            Where should the tooltip appear besides the tooltip icon.
+            It can be either top, left, right, or bottom.
+            The default value is 'bottom'.
+        """
+        tooltip_attributes = {
+            "_class": "bx--tooltip--definition bx--tooltip--a11y ",
+            "data_tooltip_definition": True,
+        }
+
+        tooltip_attributes = hg.merge_html_attrs(tooltip_attributes, attributes)
+        asst_txt_id = hg.html_id(self, "bx--tooltip--definition-id")
+
+        super().__init__(
```

Comment:
you can save two lines of code here by extending `DIV` directly insteady of `BaseElement`, but it's no big deal

Reply:
In some cases, like `InteractiveTooltip`, it doesn't come as a whole element of `<div>`, but it's separated into two parts, one for a label, and another for the tooltip itself.